L/Libtiff/build_tarballs.jl

```diff
@@ -16,7 +16,7 @@ script = raw"""
 cd $WORKSPACE/srcdir/tiff-*/
 CPPFLAGS="-I${prefix}/include"
```

Comment:
This needs to be exported
```suggestion
export CPPFLAGS="-I${prefix}/include"
```

Reply:
The build looks good with this fix, whereas my commit was entirely nonsensical based on outdated information.